Sources/Authentication/LoginSessionConfiguration.swift

```diff
@@ -39,7 +38,6 @@ public struct LoginSessionConfiguration {
                 clientID: String,
                 prefersEphemeralWebSession: Bool = true,
                 redirectURI: String,
-                nonce: String = UUID().uuidString,
```

Comment:
@DylanRJ is the removal of this value a breaking change? Eg. to any other consumers of this package? (Are there any other consumers at the moment...?) If so, should we put a `!` in the conventional commit etc?

Reply:
Yes, it's a breaking change 😄 believe you are currently the only consumer, but definitely we should `!` it; let's start as we mean to go on!